docs/device_reference.md

```diff
@@ -8,6 +8,7 @@ There are currently two devices:
 |:---:|:-------:|:-----:|:--:|:-----:|:---------:|:----:|:---------:|:----------:|
 | 2nd | 2018    | AEOBP | 5  | 10.1" | xlarge    | mdpi | WXGA 10.1 | 1280x800   |
 | 1st | 2017    | AEOKN | 5  | 7"    | large     | mdpi | ?         | 1024x600   |
+| 5th | 2019    | AEOCH | ?  | ?     | large     | ?    | ?         | 960x480    |
```

Comment:
Technically, this isn't 5th gen: I wonder if we should rename it. Maybe "Series"? Also, I had it chronological by release date so this should be at the top.